docs/articles/book/getting-started/index.md

```diff
@@ -0,0 +1,198 @@
+# Beginning Our NUnit TDD Journey
+
+In this exercise, we'll introduce you to NUnit and some NUnit concepts by doing some introductory test-driven development at the same time.
+
+## Introducing the Exercise: A Temperature Converter
+
+_This exercise comes to us via [Fadi Stephan](https://www.linkedin.com/in/fadistephan/) of [Kaizenko](https://www.kaizenko.com/) who first taught it to the author. Used here with his permission. Thanks, Fadi!_
+
+Let's imagine a scenario in which we have to convert Celsius temperatures to Fahrenheit.
+
+Before jumping in, we think a little about the problem space and some things we might know about the conversion:
+
+* We'll probably have something that converts temperatures, probably called `TemperatureConverter`.
+* We're converting Celsius to Fahrenheit, so we'll probably have a method in that called `ConvertCToF`, or similar.
+* That method will probably take in a decimal and return a decimal, e.g. `public decimal ConvertCToF(decimal celius)`.
+* We also probably know at least some of these common conversions:
+
+| Celsius Temperature | Fahrenheit Temperature |
+| ------------------- | ---------------------- |
+| 0                   | 32                     |
+| 100                 | 212                    |
+| 37                  | 98.6                   |
+| -40                 | -40                    |
+
+## Creating the Project Structure
+
+* Create a new, empty folder somewhere that you want to practice this exercise
+* Open a command prompt and go to that folder.
+* Run the following commands:
+
+```cmd
+dotnet new sln --name Conversions
+dotnet new classlib --name Conversions
+dotnet new nunit --name Conversions.Tests
+dotnet sln add .\Conversions\
+dotnet sln add .\Conversions.Tests\
+cd .\Conversions.Tests\
+dotnet add reference ..\Conversions\Conversions.csproj
+```
+
+Let's break down what this command does.
+
+* Adds a new solution named `Conversions.sln`
+* Adds a new class library called `Conversions.csproj` (this will be our production code project)
+* Adds a new test project using the `nunit` template, called `Conversions.Tests.csproj` (this will be our unit test project)
+* Adds a reference from the test project to the production code project so that we can see its contents.
+
+As for the `nunit` template, what does that do for us "out of the box"? In the `Conversions.Tess.csproj` file, we can see that a few libraries have been added for us:
+
+[!code-xml[PackageListing](~/snippets/book/getting-started/Converter.Tests/Converter.Tests.csproj#L11-L23)]
+
+* `Microsoft.NET.Test.Sdk` brings the test platform along and allows tests to be run using the `dotnet test` command. It's a good idea to add this to any .NET automated test project.
+* `NUnit` brings in the core NUnit library.
+* `NUnit3TestAdapter` surfaces NUnit3 tests for the Visual Studio / `dotnet test` runner. It essentially allows NUnit tests to be discovered.
```

Comment:
> It essentially allows NUnit tests to be discovered.

It essentially allows NUnit tests to be executed.

(Discovery may or may not be happening with the adapter, so its main focus is to handle the execution)

Reply:
Good catch, thanks 👍